tuber/client.py

```diff
@@ -443,7 +461,6 @@ class SimpleTuberObject:
     """
 
     _context_class = SimpleContext
-    _tuber_objname = None
```

Comment:
This one is actually important for SQLAlchemy-type frameworks- the __getattr__ call below accesses _tuber_objname, which does not exist if the constructor hasn't been called.

Reply:
Merge/diff hijinx and confusion on my part!